### ISSUE TITLE
Restic works but not as a YNH app

### ISSUE THREAD

**Issue by deljones** [YunoHost-Apps/restic_ynh#13]
My YunoHost server

Hardware: VPS bought online / Old laptop or computer / Raspberry Pi at home / Internet Cube with VPN / Other ARM board / …
YunoHost version: latest
I have access to my server : Through SSH | through the webadmin | direct access via keyboard / screen | …
Are you in a special context or did you perform some particular tweaking on your YunoHost instance ? : no / yes
If yes, please explain:

Description of my issue

Hello all.

Just looking to fill in the gaps with Restic. I have installed the Yunohost app. I installed it with the default time settings, changing of course the location of my external USB drive. I received the email regarding the ssh key and followed the instructions in the mail.

If my understanding is correct Restic is supposed to do a backup at 00:15 every day? I looked today and nothing has happened, If I use the CLI I can do backups/restore manually, I can do everything via the CLI with Restic with no problem what I’m trying to do now is tell the Restic app what to backup and at what time which is what I thought was in the app set up at install, and there is no config available via the app once installed.

Can anyone help me with this by any chance?

Is it best to just uninstall the app and install Restic via the CLI and create a CRON job etc?
Where do I find the YNH app of Restic log to see what has happened (so far nothing using the app)?

I have confirmed Restic works I just can’t find a way to tell it what apps and when.

Basically, I want the whole server, users, emails, apps etc backed up at 01:00

Many thanks

Dj

Sorry, I forgot to add.

I did get an email when it attempted the scheduled backup..

SUBJECT: YunoHost Restic backup log on [xxxxxxxxxxxx.co.uk](http://xxxxxxxxxxxx.co.uk/) (SUCCESS)

The rest of the email was empty. Again when I look at the repository I only have the two snapshots I did manually via the CLI
